resources/views/objects/user-dropdown.blade.php

```diff
@@ -16,50 +16,90 @@
-			title="{{ trans("users.logout._") }}"
-			href="{{ route("users.logout") }}"
-			data-method="delete"
-			data-confirm="{{ trans("users.logout.confirm") }}"
-			data-remote="1"
-		>
-			<i class="fa fa-sign-out"></i>
-		</a>
 	</div>
 
-	{{--
-	<!-- each of these is a notification inbox -->
-	@if (Auth::user()->isBAT())
-		<a href="#" class="n" id="notifications-bat">
-			<span class="badge" style="color: #fa3703 !important">bat</span>
+	<a class="avatar avatar--nav js-nav-avatar" href="#" title="{{ trans("users.show.avatar", ["username" => Auth::user()->username]) }}" style="background-image: url('{{ Auth::user()->user_avatar }}');" data-toggle="modal" data-target="#user-dropdown-modal"></a>
+@else
+	<div id="nav-user-bar">
+		<a href="#" title="{{ trans("users.anonymous.login") }}" data-toggle="modal" data-target="#user-dropdown-modal">
+			{{ trans("users.anonymous.username") }}
 		</a>
-	@endif
+	</div>
 
-	@if (Auth::user()->isGMT())
-		<a href="#" class="n" id="notifications-gmt">
-			<span class="badge green-dark">gmt</span>
-		</a>
-	@endif
+	<a class="avatar avatar--nav avatar--guest js-nav-avatar" href="#" title="{{ trans("users.anonymous.login") }}" data-toggle="modal" data-target="#user-dropdown-modal"></a>
+@endif
 
-	@if (Auth::user()->isAdmin())
-		<a href="#" class="n" id="notifications-admin">
-			<span class="badge blue-dark">admin</span>
-		</a>
-	@endif
-	--}}
+@section('user-dropdown-modal')
+	<div id="user-dropdown-modal" class="modal fade" tabindex="-1">
+		<div class="modal-dialog modal__dialog js-user-dropdown-modal__dialog">
+			@if (Auth::check()) 
+				<div class="modal-content modal-content--authenticated">
+					<div class="modal-header modal-header--authenticated" style="background-image: url('/temp-issue-35-related/bg-cover.png');">
+						<div class="modal-header__dimmer"></div>
+						<div class="modal-header__badges badges">
+							<i class='badges__badge badges__badge--small badges__badge--level'>5</i>
+							<i class='badges__badge badges__badge--small badges__badge--achievements'>5</i>
+						</div>
+						<div class="modal-header__userinfo userinfo-small">
+							<h1 class="userinfo-small__username">{{ Auth::user()->username }}</h1>
+							<a href="#/rankings/country-maybe?" class="userinfo-small__country" style="background-image: url('/images/flags/{{ "JP" }}.png');"></a>
+							<a href="#/teams/ppy" class="userinfo-small__team" style="background-image: url('/temp-issue-35-related/default-team.png');"></a>
+						</div>
+						<div class="modal-header__ranking rankinginfo-small">
+							<span class="rankinginfo-small__gamemode"><i class="fa osu fa-ctb-o"></i> #37542</span>
+							<span class="rankinginfo-small__country">Malaysia #37542</span>
+						</div>
+					</div>
+					<div class="modal-body modal-body--compartimentalized">
+						<div class="modal-body__compartment modal-body__compartment--left quick-info">
+							<h1 class="quick-info__level">Level 5</h1>
```

Comment:
Right. Line 51 is? (`<h1 class="userinfo-small__username">{{ Auth::user()->username }}</h1>`)